src/form/fields/StringField.test.tsx

```
@@ -1,11 +1,27 @@
 import { ModelContext, ModelContextProvider } from '../providers/ModelProvider';
 import { SchemaProvider } from '../providers/SchemaProvider';
 import { ROOT_PATH } from '../utils';
 import { StringField } from './StringField';
 
+const StatefulSuggestionProvider = ({
+  children,
+  getProviders,
+}: {
+  children: ReactNode;
+  getProviders: jest.Mock;
+}) => {
```

ESLint

[error] Replace `⏎··children,⏎··getProviders,⏎}:·{⏎··children:·ReactNode;⏎··getProviders:·jest.Mock;⏎` with `·children,·getProviders·}:·{·children:·ReactNode;·getProviders:·jest.Mock·`

(prettier/prettier)